art/estimators/object_detection/python_object_detector.py

```diff
@@ -214,7 +214,7 @@ def _get_losses(
                     x_grad.requires_grad = True
                 else:
                     x_grad = x[i].to(self.device)
-                    if x_grad.shape[-1] in [1, 3]:
+                    if x_grad.shape[2] < x_grad.shape[0] and x_grad.shape[2] < x_grad.shape[1]:
```

Comment:
Could we keep the channels_first variable to allow manual configuration of the channels, and if it is not supplied, infer the channel number via this logic? There could be unusual cases in which the channels are more than one of the height/width dimensions of the input.

Reply:
I agree. For this patch release we decided to keep the `channels_first` unused, but update the documentation accordingly`, with the goal not to break the estimator. I agree, there could be images with more channels than pixels in height and width. All the models supported by this estimator are expecting channels first, therefore we should update the `channels_First` argument` for ART 1.11, I have opened issue #1637.